flang/runtime/exceptions.cpp

```diff
@@ -46,7 +42,11 @@ uint32_t RTNAME(MapException)(uint32_t excepts) {
   Terminator terminator{__FILE__, __LINE__};
 
   static constexpr uint32_t v{FE_INVALID};
-  static constexpr uint32_t s{__FE_DENORM}; // subnormal
+#if __x86_64__
+  static constexpr uint32_t s{__FE_DENORM}; // nonstandard, not a #define
```

Comment:
If `__FE_DENORM` isn't a macro and isn't standard, how can we rely on it being defined? This breaks the build on FreeBSD with the base system llvm-18 based clang compiler.